### PR TITLE
Allow usb0 interface for SSH access through rndis

### PR DESCRIPTION
**Pull request by DylanVanAssche**

### CODE REVIEW

File: sparse/etc/usb-moded/dyn-modes/99-developer_mode-usb.ini [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[mode]
+name = developer_mode
+module = none
+network = 1
+network_interface = usb0
+appsync = 1
+
+[options]
+sysfs_path = /sys/class/android_usb/android0/functions
```
Comment (piggz) [inline sparse/etc/usb-moded/dyn-modes/99-developer_mode-usb.ini:9]: are these lines needed?

Comment (DylanVanAssche) [reply sparse/etc/usb-moded/dyn-modes/99-developer_mode-usb.ini:9]: Without them, it doesn't work anymore

File: sparse/etc/usb-moded/dyn-modes/99-developer_mode-usb.ini [hunk 2]
```diff
@@ -0,0 +1,16 @@
+[mode]
```
Comment (piggz) [inline sparse/etc/usb-moded/dyn-modes/99-developer_mode-usb.ini:1]: What about calling it developer_mode-native ?

Comment (DylanVanAssche) [reply sparse/etc/usb-moded/dyn-modes/99-developer_mode-usb.ini:1]: hmmm good idea, will do that as soon as the CI works again

**Pull request merged**